experiments/experiment_prototype.py

```diff
@@ -1,385 +1,1111 @@
+    
+    DESCRIPTION OF SLICE KEYS
+
+slice_id : The ID of this object. An experiment can have multiple objects.
+cpid: The ID of the experiment, consistent with existing radar control programs.
+txantennas: The antennas to transmit on, default is all antennas given max number from config.
+rx_main_antennas: The antennas to receive on in main array, default = all antennas given max number from config.
+rx_int_antennas : The antennas to receive on in interferometer array, default is all antennas given max number from config.
+pulse_sequence: The pulse sequence timing, given in quantities of mpinc, for example normalscan = [0, 14, 22, 24, 27, 31, 42, 43]
+mpinc: multi-pulse increment in us, Defines minimum space between pulses.
+pulse_shift: Allows phase shifting between pulses. Built in for a capability Ashton would like to use within a pulse sequence. Default all zeros for all pulses in pulse_sequence.
+pulse_len: length of pulse in us. Range gate size is also determined by this.
+nrang: Number of range gates.
+frang: first range gate, in km
+intt: duration of an integration, in ms. (maximum)
+intn: number of averages to make a single integration, if intt = None.
+beam_angle: list of beam directions, in degrees off azimuth. Positive is E of N. Array length = number of beams.
+beam_order: beamnumbers written in order of preference, one element in this list corresponds to one integration period. Can have list within lists. a beamnubmer of 0 in this list gives us beam_angle[0] as a direction.
+scanboundflag: flag for whether there is a scan boundary to wait for in order to start a new scan.
+scanbound: time that is alloted for a scan before a new scan boundary occurs (ms).
+clrfrqrange: range for clear frequency search, should be a list of length = 2, [min_freq, max_freq] in kHz.
+txfreq: transmit frequency, in kHz. Note if you specify clrfrqrange it won't be used.
+rxfreq: receive frequency, in kHz. Note if you specify clrfrqrange or txfreq it won't be used. Only necessary to specify if you want a receive-only slice.
+acf: flag for rawacf and generation. Default True.
+xcf: flag for cross-correlation data. Default True
+acfint: flag for interferometer autocorrelation data. Default True.
+wavetype: default SINE. Any others not currently supported but possible to add in at later date.
+seqtimer: timing in us that this object's sequence will begin at, after the start of the sequence.
 
-    def __init__(self, cponum, cpid):
+Should add:
 
-        self.cpid=cpid # REVIEW #40 use spaces around '=' signs - it's easier to read. Follow PEP 8 style guide. This applies to all
-        # Unique ID for each new cp.
-        self.cponum=cponum
-        # Number of CPObjects in this program. # REVIEW #38 These comments are superfluous since the docstring tells all you need to know
-        cpo_list=[]
+scanboundt : time past the hour to start a scan at ?
+    
+        Traditionally beams have been 3.75 degrees separated but we don't refer to them as beam 
```

Comment:
Kevin seems to remember different numbers for degree separation and antenna separation. Maybe just clarify for him that these are correct.